templates/service/kosher.txt

```diff
@@ -0,0 +1,24 @@
+[hr][center]
+[small]With gods help
+[table][row][cell]Kosher Department[cell][solcrest][cell]Sector Rabbinate[/table][/small]
```

Comment:
Who would be using this? The Torch is in mostly uncharted space, so there's probably not a local rabbinate. If the Chaplain would, then maybe this language should be changed.

Reply:
Yeah, that's the idea. The Chaplain would use this.
The content was taken from IRL kosher certificates, so the language is from there.
Perhaps should ask a lore dev about how this would be structured.

Reply:
```suggestion

```
Let's cut this to make it able to be used for either purpose.